src/Components/Publishing/ToolTip/ToolTip.tsx

```diff
@@ -6,12 +8,16 @@ import { GeneToolTipContainer } from "./GeneToolTip"
 
 interface Props extends React.HTMLProps<HTMLDivElement> {
   arrowLeft?: string
-  entity: object
   orientation?: string
   model: string
   onMouseEnter?: any
   onMouseLeave?: any
   positionLeft?: number
+  /**
+   * Having to tpye this opaque data manually is an exceptional case, because of
+   * how this component setup doesn’t strictly follow a Relay tree.
+   */
+  entity: { " $fragmentRefs": ArtistToolTip_artist$ref & GeneToolTip_gene$ref }
```

Comment:
Little typo, but otherwise would this edgecase be appropriate to explain in #frontend?

Reply:
Fixed the typo, nice spot! 🕵️ 

Re the edge-case, I dunno, I’d rather people get used to using it in general first. It’s good that you at least know of it too now.